### DISCUSSION TITLE
how can i change back to the default theme

### DISCUSSION THREAD

**Discussion by nglong05** [hyprwm/Hyprland#8725] [Q&A]
I just got Hyprland working and really liked the default theme that appeared initially. But i selected a different theme from a list in the status bar, and now the new status bar doesn’t include the option to switch themes back. How can I restore the default theme?

**Comment by izmyname** [comment]
Hyprland doesn't have a status bar and a concept of 'theme', besides rgba colors. You probably installed someone's dotfiles.
